wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -1577,6 +1671,29 @@ private function preprocess_string_literal( $value ) {
 		return $value;
 	}
 
+	private function preprocess_like_expr( &$token ) {
+		/*
+		 * This code handles escaped wildcards in LIKE clauses.
+		 * If we are within a LIKE experession, we look for \_ and \%, the
+		 * escaped LIKE wildcards, the ones where we want a literal, not a
+		 * wildcard match. We change the \ escape for a ~ character,
```

Comment:
Is `~` a valid escape character in SQLite?

Reply:
Bogus comment, now fixed. I'm using the ASCII SUB (substitute, 0x1a) character. It tests out fine.